test/apiTest.js

```diff
@@ -150,36 +179,37 @@ describe('POST /cats', function () {
     }) 
 
     it('respond with 400 invalid parameter`', function (done) {
-        request(app)
+        request(appTest)
             .post('/cats')
             .send(invalidParams)
             .set('Accept', "text/html; charset=utf-8")
             .expect('Content-Type', "text/html; charset=utf-8")
-            .expect(400, "Invalid parameter: 'five'", done)
-    })
+            .expect(400,'Invalid ageInYears parameter: "five"', done)
+    }) 
 })
+// Error: expected 400 "Bad Request", got 500 "Internal Server Error"
 
 describe('GET /breeds', function () {
     it('respond with json object containing a list of all breeds', function (done) {
-        request(app)
+        request(appTest)
             .get('/breeds')
-            .set('Accept', 'application/json')
+            .set('Accept', 'appTestlication/json')
```

Comment:
there are a couple of similar typos in the tests

Reply:
haha my first dabblings in the Find & Replace tool.